src/commands/projects/create/nosql.ts

```diff
@@ -28,6 +28,16 @@ export default class NosqlCommand extends AbstractProjectCreateCommand {
       exclusive: ['databaseConnectionURL'],
       required: false,
     }),
+    javascript: flags.boolean({
```

Comment:
Quick question, should we really consider the javascript file ? If we opt for the following: 
- No flag is Javascript
- Typescript flag is for typescript

--> We could get ride of it. Wdyt ?

Reply:
I think you did this to get the proposal from the prompts... I'm questioning the necessity of it.

Reply:
yeah, but IMO having it in the command shows to the users that there are other language options, so I'd be in favor of having the option.